Expensify/App
# Fix modal on web doing wacky behavior

<Assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

<Explanation of the change or anything fishy that is going on>
cc @tgolen @roryabraham @alex-mechler 

### Fixed Issues
https://github.com/Expensify/Expensify/issues/146479

### Tests
1. Open web/mobile
2. Open an image modal by clicking on the preview
3. Close the image modal
3. Switch chats and make sure you don't go on an adventure (you see the correct chat and you aren't scrolling everywhere)

### Screenshots
<Add any necessary screenshots for all platforms if your change added or updated UI>
